edit-post/store/constants.js

```diff
@@ -9,3 +9,9 @@ export const BREAK_LARGE = parseInt( breakpointsScssVariables.breakLarge );
 export const BREAK_MEDIUM = parseInt( breakpointsScssVariables.breakMedium );
 export const BREAK_SMALL = parseInt( breakpointsScssVariables.breakSmall );
 export const BREAK_MOBILE = parseInt( breakpointsScssVariables.breakMobile );
+
+/**
+ * Other constants used throughout the editor
+ */
+
+export const BLOCK_REORDER = 'BLOCK_REORDER';
```

Comment:
Need to review this constant and whether this is the right place for it (or use `editor/store/selectors` instead).

Reply:
What _is_ this constant? I'm sure I'm about to find out in the course of my own review, but we should be considerate of future readers of just this file alone and include a DocBlock explaining its purpose.

Reply:
>(or use `editor/store/selectors` instead).

What relation to selectors does this constant have?

I'm wondering if we ought to just take the more direct approach and inline this as a string, `type: 'block'`, and call it a day.

Reply:
This particular constant is not currently used (I switched to a selector instead - but the question/problem is the same - neither is a good option right now). The constant is added to `dataTransfer` in `onDragStart` so that the `onDrop` handler in `BlockDropZone` only handles block reordering. We could potentially drop something else in a block's drop zone (as it will be the case if we extend drag & drop to reorder other lists e.g. image galleries). 

With the latest changes (moved the drag logic to a separate HOC), the constant is particular to the `withDragging` HOC, so it might be best to move it there now. Not sure what the best way is to export it from there. Maybe pass this and any other future constant as a property to the component being wrapped with the HOC (just as I pass `onDragStart` and `onDragEnd`)?

Reply:
> I'm wondering if we ought to just take the more direct approach and inline this as a string, type: 'block', and call it a day.

The reason for it being here is to avoid processing in the block onDrop handling anything else that is dragged to be reordered on the page. It is `BLOCK_REORDER` here and will be `IMAGE_GALLERY_REORDER` next if we choose to get there with this logic. So we need a single source to provide the the constants that the component to be dragged will use to parameterise the drag init logic (which sets it to dataTransfer).

Reply:
> What relation to selectors does this constant have?

Nothing really - it's just I've seen other constants being exported there. These constants are best be delivered via the HOC IMO, but I need to review their use to make sure my logic I describe is correct and still relevant. :-)

Reply:
> I'm wondering if we ought to just take the more direct approach and inline this as a string, type: 'block', and call it a day.

This might be the best solution after all. After all the value is only passed to the `onDragStart` handler to be added to `dataTransfer` - and that's used later by the block's drop zone alone. So it can be anything block-specific.